### ISSUE TITLE
Differences between alternative Drone Helm plugins

### ISSUE THREAD

**Issue by lukassup** [kubeciio/helm#2]
Are there any notable differences between [`ipedrazas/drone-helm` plugin](https://github.com/ipedrazas/drone-helm) and this one?

**Comment by so0k** [comment]
Agreed, ipedrazas/drone-helm is currently the established drone helm plugin, I wonder what additional features this provides

**Comment by metalmatze** [comment]
Hey!
Sorry for letting you wait so long.

Right now you're correct, the plugin has no notable differences with ipedrazas/drone-helm plugin.
We're adding some more features in the next weeks. One is the ability to define the helm version like `version: v2.9.1` and then fetch the correct version. Other features are planned too.
